Framework/Algorithms/inc/MantidAlgorithms/ReflectometryReductionOneAuto2.h

```diff
@@ -82,6 +78,10 @@ class DLLExport ReflectometryReductionOneAuto2
   std::tuple<API::MatrixWorkspace_sptr, std::string, std::string>
   getPolarizationEfficiencies();
   void applyPolarizationCorrection(std::string const &outputIvsLam);
+  API::MatrixWorkspace_sptr getFloodWorkspace();
+  void applyFloodCorrection(API::MatrixWorkspace_sptr const &flood,
+                            std::string const &prop);
```

Comment:
Just some lines further someone wrote:
const std::string &propertyName
why changing the style and name here?